From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Leif Lindholm <leif.lindholm@linaro.org>
Date: Wed, 14 Nov 2018 19:29:16 +0000
Subject: [PATCH] include/grub/file.h: Add device tree file type

The API change of grub_file_open() for adding verifiers did not include
a type for device tree blobs. Add GRUB_FILE_TYPE_DEVICE_TREE_IMAGE to
the grub_file_type enum.

Signed-off-by: Leif Lindholm <leif.lindholm@linaro.org>
Reviewed-by: Daniel Kiper <daniel.kiper@oracle.com>
(cherry picked from commit 7453c2cc32525a5eebe3b268433d0dfc73622917)
Signed-off-by: Daniel Axtens <dja@axtens.net>
---
 include/grub/file.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/grub/file.h b/include/grub/file.h
index 19dda67f6..9aae46355 100644
--- a/include/grub/file.h
+++ b/include/grub/file.h
@@ -69,6 +69,8 @@ enum grub_file_type
 
     GRUB_FILE_TYPE_EFI_CHAINLOADED_IMAGE,
 
+    GRUB_FILE_TYPE_DEVICE_TREE_IMAGE,
+
     /* File holding signature.  */
     GRUB_FILE_TYPE_SIGNATURE,
     /* File holding public key to verify signature once.  */
